### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by rickbacci**
@joshcheek @selfup @jbrr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# TwoCentsWorth
```
Comment (jbrr) [inline README.md:1]: I like the new name!

File: server.js [hunk 2]
```diff
@@ -25,22 +25,17 @@ app.get('/', function(request, response) {
...
 app.post('/polls', function(request, response) {
-  var pollType    = request.body.polltype;
-  var status      = 'open';
   var adminString = randomString();
   var voterString = randomString();
-  var question    = request.body.question;
-  var choices     = request.body.poll.choices;
-  var endTime     = request.body.endtime;
 
   var poll = {
```
Comment (jbrr) [inline server.js:31]: This is nice, those variable assignments weren't really necessary.

File: server.js [hunk 3]
```diff
@@ -62,30 +57,33 @@ app.post('/polls', function(request, response) {
...
 
-
-    poll.choices = poll.choices.split(',')
     response.render('poll', { poll: poll })
 
   });
 
 });
 
+function formatData(poll, host) {
```
Comment (jbrr) [inline server.js:74]: I like this too, this is the abstraction that Steve mentioned last night?

**Comment by jbrr** [comment]
I think this looks really good. I like pulling the random string method into a separate file to make it more modular, it just seems like good practice in general. What's your next step?

File: README.md (hunk 1 above)
Comment (selfup) [inline README.md:1]: Yea! Makes it more _fun_

**Comment by selfup** [comment]
@jbrr I agree on making it more modular. I did the same thing!

Way to go @RickBacci

**Comment by selfup** [comment]
@RickBacci Is your next step to hook up the sockets?

File: server.js (hunk 3 above)
Comment (rickbacci) [inline server.js:74]: Kinda. I want to pull out Poll eventually, but after talking to Meeka I am going to work on tying the voting together to start tallying votes

**Comment by rickbacci** [comment]
As a voter, when I visit the voter specific link, then I will see the poll name and options

That's my next ticket.

**Comment by rickbacci** [comment]
@selfup yeah jade is cool, was a ton of googling last night, but i thing worth it.

**Comment by selfup** [comment]
Nice looking forward to it!

**Pull request merged**